File: Source/ContextMaker.m
//
//  ContextMaker.m
//  RSBarcodes
//
//  Created by R0CKSTAR on 11/22/16.
//  Copyright (c) 2016 P.D.Q. All rights reserved.
//

#import "ContextMaker.h"

@implementation ContextMaker

+ (CIContext*) make
{
   return [CIContext contextWithOptions:nil];
}

@end

File: Source/ContextMaker.swift
//
//  ContextMaker.swift
//  RSBarcodes
//
//  Created by Alexey Korolev on 11.10.2019.
//  Copyright © 2019 P.D.Q. All rights reserved.
//

import UIKit

final class ContextMaker {
    static func make() -> CIContext {
        return CIContext(options: nil)
    }
}
